app/controllers/app_groups_controller.rb

```diff
@@ -245,16 +260,37 @@ def toggle_redact_status
     end
   end
 
+  def toggle_app_group_status
+    from_status = @app_group.status
+    @app_group.status = params[:toggle_app_group_status] == 'true' ? :ACTIVE : :INACTIVE
+    @app_group.save!
+
+    audit_log :toggle_app_group_redact_status, { "from_status" => from_status, "to_status" => @app_group.redact_status }
+
+    if params[:app_group_id]
+      app_group = AppGroup.find(params[:app_group_id])
+      redirect_to app_group_path(app_group)
+    else
+      redirect_to app_groups_path
+    end
```

Comment:
we need to do a deletion on this one right?

Reply:
@clavinjune you will handle this based on yesterday sync right ?